app/decorators/physical_server_decorator.rb

```diff
@@ -23,6 +23,8 @@ def quadicon
         :tooltip  => health_state
       }
     }
+    icon[:middle] = { :fileicon => '100/shield.png' } if get_policies.present?
```

Comment:
I don't really like this. Can we perhaps rewrie it to this:
```ruby
def quadicon
  {
    :top_left => {:text => (host ? 1 : 0)},
    #...
    :middle => -> {{ :fileicon => '100/shield.png' } if get_policies.present?}[]
  }
end
```

Reply:
We were debating about this with @martinpovolny and @tumido, I wanted to use a conditional merge, but as I learned *Ruby is not a functional language, so it's fine to do it as it is*.

Reply:
What am I doing on this PR? Come one @karelhala, this is Ruby not JavaScript. Readability matters here. :shipit: